src/Compilers/CSharp/Portable/Symbols/SubstitutedNamedTypeSymbol.cs

```diff
@@ -229,7 +229,18 @@ public sealed override ImmutableArray<Symbol> GetMembers()
 
             if (IsTupleType)
             {
-                builder = AddOrWrapTupleMembers(builder.ToImmutableAndFree());
+                var existingMembers = builder.ToImmutableAndFree();
+                var fieldsToRemove = PooledHashSet<Symbol>.GetInstance();
```

Comment:
>fieldsToRemove [](http://example.com/codeflow?start=20&length=14)

I think the name of the local and the parameter of ```MakeSynthesizedTupleMembers``` is confusing. Consider renaming to "replacedFields" or something similar.